draft-packages/empty-state/KaizenDraft/EmptyState/README.mdx

```diff
@@ -26,37 +26,35 @@ import Dont from "docs-components/Dont"
 ## Options
 
 *   Mood:
-    *   Default
     *   Positive
     *   Informative
-    *   Cautionary
+    *   Action
+    *   Neutral
     *   Negative
-*   Border style:
-    *   Solid
```

Comment:
None of our empty states have border styles anymore. Can you remove this reference altogether, 
@dlimiter

Reply:
Ah yep that's the delta – it's removed 👍